README.md

```diff
@@ -1,17 +1,16 @@
 # gluster-kubernetes
 
-## Dynamic, persistent storage for Kubernetes with GlusterFS
+## GlusterFS + heketi on Kubernetes
```

Comment:
Apparently we're not the home for dynamic, persistent storage since we don't host the GlusterFS volume plugin for Kubernetes nor heketi itself, and those two things are what provide the dynamic provisioning and persistent volume support for GlusterFS. I can tack on the phrase "Hyper-converged" at the beginning, I suppose?